packages/design/src/Typography.mdx

```diff
@@ -0,0 +1,104 @@
+---
+name: Typography
+menu: Design
+route: /typography
+---
+
+import { Playground } from "docz";
+import { Heading } from "@jobber/components/Heading";
+import { Text } from "@jobber/components/Text";
+import { Emphasis } from "@jobber/components/Emphasis";
+import { Content } from "@jobber/components/Content";
+import { Button } from "@jobber/components/Button";
+import { Checkbox } from "@jobber/components/Checkbox";
+
+# Typography
```

Comment:
@hosseintalebi I don't want to get into the specific values/tokens throughout this doc, as I think that's more of a concern for the typography component(s); my thinking is more to describe the main components consumers will use, and the rationale of how we arrived at the values those components are built with.